config.js

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
That file is already in base pb. I dont want to push back an extra file we dont need.